### ISSUE TITLE
BUG: AWS Lambda functions are timing out when they shouldn't.

### ISSUE THREAD

**Issue by dionjwa** [Autodesk/cloud-compute-cannon#146]
tl;dr Scaling might be broken.

The AWS lambda functions are written to execute as fast as possible then exit. However, looking at the logs indicates that at least the scaleUp function is timing out, rather than cleanly executing:

E.g. a scaleUp lambda starts at this time:

![image](https://user-images.githubusercontent.com/260221/30499576-ea7c1668-9a0f-11e7-9b0b-1905b33c801e.png)

But then times out:
![image](https://user-images.githubusercontent.com/260221/30499604-036ca0d4-9a10-11e7-91b3-6d8ae87c83b4.png)

The last log occurs 5 seconds after the start, so there is 25 of no logs before the timeout. The code indicates it should exit quickly, and return a valid string, but there is no record of that in the logs.

**Comment by dionjwa** [comment]
Fixed by https://github.com/Autodesk/cloud-compute-cannon/commit/60194ca7ef41a42a0acd9bee09122c517bb0ced9
